### ISSUE TITLE
Asset details in event details can not be shown

### ISSUE THREAD

**Issue by mheyen** [opencast/opencast#1989]
**Describe the bug**
When I open the Asset details of an Event in the Admin UI, I encounter an error. 
![image](https://user-images.githubusercontent.com/23011761/96836911-edcf0b00-1445-11eb-98fd-412dd8a66562.png)

When I click on `Details`, the details are not displayed. 
![image](https://user-images.githubusercontent.com/23011761/96836975-fe7f8100-1445-11eb-9568-6da117bbc8ae.png)

**To Reproduce**
_(at stable.opencast.org & develop.opencast.org)_
1. Go to `Events`
2. Click on `Open asset details` of an Event
3. Click on `Details` on any Asset type
4. No details are shown
5. This error shows up on the console (sometimes only after closing the `Events details`):
```
Error: [$injector:unpr] Unknown provider: undefinedProvider <- 
https://errors.angularjs.org/1.8.0/$injector/unpr?p0=undefinedProvider%20%3C-%20
    Angular 5
    loadSubNavData resourceModalService.js:166
    Angular 5
    newSchedule resourceModalService.js:43
    loadSubNavData resourceModalService.js:178
    Angular 5
    newSchedule resourceModalService.js:43
    loadSubNavData resourceModalService.js:178
    Angular 5
    newSchedule resourceModalService.js:43
    openSubTab resourceModalService.js:140
    fn bower.42e5624c.js line 1 > Function:4
    Angular 4
    jQuery 8
    Angular 17
    show modalService.js:123
    Angular 6
angular.js:15635:40
```

**Environment Information**
 - OS: Ubuntu 20
 - Browser Firefox
 - Opencast version: r/10.0 (develop.opencast.org, 22.10.2020) & r/9.0 (stable.opencast.org, 22.10.2020)

**Comment by lkiesow** [comment]
Stealing this since I've fixed it.

**Comment by lkiesow** [comment]
Actually, this is a duplicate of #1966 
I did remember correctly that I created the issue after all :grin:
